pkg/op-forum-agg/op_forum_agg/src/sync/threads.py

```diff
@@ -77,22 +95,40 @@ def transform(self, data):
             internal_category_id = get_category_by_external_id(
                 categories, thread.rawData["category_id"]
             )
+
+            all_text = concatenate_strings(
+                about_match.group(1) if about_match else "",
+                first_post_match.group(1) if first_post_match else "",
+                reaction_match.group(1) if reaction_match else "",
+                overview_match.group(1) if overview_match else "",
+                tldr_match.group(1) if tldr_match else "",
+                classification_match.group(1) if classification_match else "",
+            )
+            read_time = estimate_reading_time(all_text)
+
             forum_post_data = {
-                "external_id": thread.external_id,
+                "external_id": thread.externalId,
                 "title": thread.rawData["title"],
                 "url": thread.url,
-                "lastPostedAt": thread.rawData["last_posted_at"],
-                "externalId": thread.external_id,
-                "category": internal_category_id,
+                "category_id": internal_category_id,
                 "about": about_match.group(1).strip() if about_match else "",
-                "firstPost": (
+                "first_post": (
                     first_post_match.group(1).strip() if first_post_match else ""
                 ),
                 "reaction": reaction_match.group(1).strip() if reaction_match else "",
                 "overview": overview_match.group(1).strip() if overview_match else "",
                 "tldr": tldr_match.group(1).strip() if tldr_match else "",
                 "username": fisrt_thread_post.rawData["username"] or "",
-                "displayUsername": fisrt_thread_post.rawData["display_username"] or "",
+                "display_username": fisrt_thread_post.rawData["display_username"] or "",
+                "raw_forum_post_id": thread.id,
+                "classification": (
+                    classification_match.group(1).strip()
+                    if classification_match
+                    else ""
+                ),
+                "last_activity": thread.rawData["last_posted_at"],
```

Comment:
is this the time when the thread had the last response? would not create the indirection in naming as `activity` as this could also be understood comprise e.g. reactions/likes to the posts